src/test/java/com/salesforce/storm/spout/dynamic/retry/DefaultRetryManagerTest.java

```diff
@@ -539,8 +539,10 @@ public void testNextFailedMessageToRetryWithExpiring() {
 
         // Now advance time by exactly expectedMinRetryTimeMs milliseconds
         retryManager.setClock(
-            Clock.fixed(Instant.ofEpochMilli(FIXED_TIME + (long) (expectedMinRetryTimeMs * expectedDelayMultiplier)), ZoneId.of("UTC"))
-        );
+            Clock.fixed(
+            Instant.ofEpochMilli(FIXED_TIME + (long) (expectedMinRetryTimeMs * Math.pow(expectedDelayMultiplier, 0))),
+            ZoneId.of("UTC")
```

Comment:
I know the old test did this as well, but fwiw I think you can do `ZoneOffset.UTC`